apps/web/app/(auth-pages)/sign-in/page.tsx

```
@@ -97,21 +96,33 @@
 					</Button>
 
 					{isNotRegistered && (
-						<div className="text-yellow-600">
+						<div className="text-yellow-600" role="alert" aria-live="assertive" tabIndex={0}>
 							User not registered. Please Sign Up first.
 						</div>
 					)}
 
 					{authError && !isNotRegistered && (
-						<div className="text-red-600">
+						<div className="text-red-600" role="alert" aria-live="assertive" tabIndex={0}>
```

Biome (1.9.4)

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)